### DISCUSSION TITLE
How can I turn off automatic translation of any fields?

### DISCUSSION THREAD

**Discussion by bluej4ker** [orchidsoftware/platform#2748] [Q&A]
For example, when I create a UserListScreen, the title of this class changes to Laravel's default language. How can I turn this off?

**Comment by c-gross** [comment]
The translation for the page title happens in vendor/orchid/platform/resources/views/layouts/base.blade.php
`@section('title', e(__($name)))`

But if you override that file and remove the translation call (`@section('title', e(__($name))) -> @section('title', e($name))`) no page title will be translated anymore.
